front/src/modules/companies/components/AddPersonToCompany.tsx

```diff
@@ -47,14 +56,19 @@ export const AddPersonToCompany = ({
 }) => {
   const [isDropdownOpen, setIsDropdownOpen] = useState(false);
   const [isCreationDropdownOpen, setIsCreationDropdownOpen] = useState(false);
-  const [username, setUsername] = useState(defaultUsername);
   const [updatePerson] = useUpdateOnePersonMutation();
   const [insertOnePerson] = useInsertOnePersonMutation();
   const { refs, floatingStyles } = useFloating({
+    open: isDropdownOpen,
     placement: 'right-start',
-    middleware: [flip(), offset({ mainAxis: -20, crossAxis: 25 })],
+    middleware: [flip(), offset({ mainAxis: 30, crossAxis: 0 })],
   });
 
+  const hotkeyCloser = useCallback(() => {
```

Comment:
Please start your function names with a verb, here the appropriate naming would be `handleEscape`. Also no need to memoize this simple handler.

Reply:
Done